app/src/org/commcare/android/fragments/InstallAppSelectFragment.java

```diff
@@ -23,10 +23,10 @@
  *
  * @author Daniel Luna (dluna@dimagi.com)
  */
-public class SetupInstallFragment extends Fragment {
+public class InstallAppSelectFragment extends Fragment {
```

Comment:
Nitpicking, but this makes it sound like you're choosing which app to install. Maybe SelectInstallModeFragment?